devel/meson/Portfile

```diff
@@ -4,30 +4,29 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        mesonbuild meson 0.59.3
+github.setup        mesonbuild meson 0.62.1
 revision            0
 
 github.tarball_from releases
 license             Apache-2
 categories          devel python
 maintainers         {soap.za.net:git @SoapZA} openmaintainer
-platforms           darwin
```

Comment:
you can leave this as it is

Reply:
@kencu Okay. Just other people kept telling me in other tickets to remove this line, as it is a default now. So I removed it here too.

Reply:
oh, I see. Sorry, didn’t know that.

In general, when doing a version update PR, you’ll be best off if you leave everything else alone…cleanups, whitespace changes, rewording comments, etc all should be in some separate PR if they need doing.